app/helpers/frontend/button_helper.rb

```diff
@@ -24,7 +24,8 @@ def generate_organization_report_button(id)
       button_tag("Generate Reports", data: {reflex: "click->OrganizationReportsReflex#generate", id: id, controller: "organization-reports"}, class: "ml-1 btn btn-primary", style: "height:2.5rem;")
     end
 
-    def layout_button(link: "#", icon: nil, add_class: "", target: "", title: nil, admin: false, method: :get, confirmation_message: nil, permissions: true)
+    def layout_button(link: "#", icon: nil, add_class: "", target: "", title: nil, admin: false, method: :get, confirmation_message: nil, permissions: true, display: true)
```

Comment:
This is getting a bit unwieldy. Permissions and display function the same. I like display over permissions. Could we remove permissions and replace it with display in the views? It isn’t used much.

Reply:
I'm not really up to date on what `layout_button` is doing but agree that `permissions: true` is ambiguous. `display:true` is certainly more clear about intention and behavior.